airbyte-integrations/connectors/source-s3/metadata.yaml

```diff
@@ -30,14 +30,23 @@ data:
   releaseStage: generally_available
   releases:
     rolloutConfiguration:
-      enableProgressiveRollout: false
+      enableProgressiveRollout: true
     breakingChanges:
       4.0.0:
         message: UX improvement, multi-stream support and deprecation of some parsing features
         upgradeDeadline: "2023-10-05"
       4.0.4:
         message: Following 4.0.0 config change, we are eliminating the `streams.*.file_type` field which was redundant with `streams.*.format`
         upgradeDeadline: "2023-10-18"
+      5.0.0:
+        # Technically, this may not actually be a breaking change because the
```

Comment:
Exactly. Let's avoid making this breaking.

Reply:
Rationale: every breaking change would mean stopped syncs. We don't want to stop syncs.

Reply:
100% agreed. And I think we can without issues. (See other comments above.)